template.tpl

```diff
@@ -1380,8 +1418,17 @@ const onsuccess = () => {
       break;
 
     case 'track':
```

Comment:
Extract this as a function and do more manual tests.
<img width="726" alt="Screenshot 2024-05-01 at 10 07 11 AM" src="https://github.com/amplitude/amplitude-browser-sdk-gtm-template/assets/13028329/84b10a1e-8cf4-47f4-957b-436099bc1be4">
<img width="691" alt="Screenshot 2024-05-01 at 10 07 24 AM" src="https://github.com/amplitude/amplitude-browser-sdk-gtm-template/assets/13028329/d438ee9a-d64a-440c-8429-8b58760b5eea">
<img width="753" alt="Screenshot 2024-05-01 at 10 07 37 AM" src="https://github.com/amplitude/amplitude-browser-sdk-gtm-template/assets/13028329/e50828fd-040a-4063-a2a6-d57a896da8be">